src/java/org/apache/cassandra/io/sstable/SSTable.java

```diff
@@ -81,7 +83,7 @@
     public static final int TOMBSTONE_HISTOGRAM_TTL_ROUND_SECONDS = Integer.valueOf(System.getProperty("cassandra.streaminghistogram.roundseconds", "60"));
 
     public final Descriptor descriptor;
-    protected final Set<Component> components;
+    public final Set<Component> components;
```

Comment:
This seems to have been named public for its usage in `SAITester#verifySSTableComponents`. However, the class has a `SSTable#getComponents` method that can be used instead of exposing the attribute.